kedro/io/core.py

```diff
@@ -101,6 +101,9 @@ class AbstractDataSet(abc.ABC):
         >>>         return dict(param1=self._param1, param2=self._param2)
     """
 
+    DEFAULT_LOAD_ARGS = {}
```

Comment:
Is this something we can leave to the classes extending `AbstractDataSet`?

Reply:
Oh, sorry, I forgot to take this out. 🤦‍♂